resources/assets/less/bem/beatmapset-badge.less

```diff
@@ -12,23 +12,29 @@
   white-space: nowrap;
   .link-plain();
 
+  --colour: hsl(var(--hsl-c1));
+  --colour-hover: var(--colour);
+  color: var(--colour);
+
+  .link-hover({
+    color: var(--colour-hover);
+  });
```

Comment:
this doesn't seem to do anything (and apparently isn't working before the change, either?) 🤔